.github/dependabot.yml

```diff
@@ -99,6 +99,12 @@ updates:
       interval: weekly
     open-pull-requests-limit: 20
 
+  - package-ecosystem: nuget
+    directory: /src/OpenTelemetry.AutoInstrumentation.Native
+      day: sunday
```

Comment:
Candidate for separate PR - change the interval from `weekly` to `daily`. 

Benefits: We will reduce number of new PRs in Sunday/Monday morning. It should be easier to manage it.